test/passing/break_before_func.ml

```diff
@@ -0,0 +1,195 @@
+[@@@ocamlformat "break-before-func=false"]
+
+;;
+List.fooo (fun x ->
+    xxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxxxxxxxxxxx
+      xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxxxxxxxxxxx
+  )
```

Comment:
Just to check, is it intended that this paren gets bumped to its own line, when there are no following arguments? What do you think @samoht ?